server/routes/api.js

```diff
@@ -574,11 +574,15 @@ router.post('/messages', isLoggedIn, function(req, res, next) {
         res.status(200);
         return res.send('Ignoring filtered');
       }
+
+      // logger.main.error('Say hello to API.JS - IS RAW GEOLOCATION HERE?! Following: ' + JSON.stringify(data));
         var address = data.address || '0000000';
         var message = data.message || 'null';
         var datetime = data.datetime || 1;
         var timeDiff = datetime - dupeTime;
         var source = data.source || 'UNK';
+        var raw_geolocation = data.raw_geolocation || 0;
```

Comment:
The whole point of the plugin system is not to clutter up API.js from extra features like this not everybody is going to use. 
Plugins shouldn't touch API.js in my opinion